.github/workflows/edgar-feed.yaml

```diff
@@ -0,0 +1,62 @@
+name: Unit Tests
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    working-directory: edgar-feed
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  clippy:
+    name: Clippy
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
Could you include the `name` field as well? Nice to have a brief contextual descriptor for debugging.

Reply:
done